cylc/flow/network/scan.py

```diff
@@ -164,8 +165,10 @@ def scan_many(items, methods=None, timeout=None, ordered=False):
         try:
             msg = await client.async_request(method)
         except ClientTimeout as exc:
+            LOG.exception(f"ClientTimeout-reg:{reg}|host:{host}|port:{port}")
```

Comment:
(re-doing a deleted change suggestion, to keep under 80-char limit)

```suggestion
            LOG.exception(
                     f"ClientTimeout: name:{reg}, host:{host}, port:{port}")
```

Reply:
(We don't use `|` as a separator in message strings anywhere else).

Reply:
(otherwise all good now)

Reply:
Done!